lib/Bio/EnsEMBL/DataCheck/Test/DataCheck.pm

```diff
@@ -40,7 +40,7 @@ our @EXPORT  = qw(
   is_rows cmp_rows is_rows_zero is_rows_nonzero 
   row_totals row_subtotals
   fk denormalized denormalised
-  has_data
+  has_data get_data
```

Comment:
This is not the right place for such a method. This module is for tests, general purpose methods should go in Utils.pm. But in any case, I don't think this method is required; a range of methods for fetching data already exist in the core API, and those are the recommended approach for datachecks: https://www.ebi.ac.uk/seqdb/confluence/display/GTI/Everything+you+wanted+to+know+about+datachecks%2C+but+were+too+afraid+to+ask#Everythingyouwantedtoknowaboutdatachecks,butweretooafraidtoask-SQLHelpermethods